bot.py

```diff
@@ -1111,10 +1155,13 @@ def get_version_number(text):
         str: The version if it parsed correctly
     """
     pattern = re.compile(VERSION_RE)
+    hash_pattern = re.compile(COMMIT_HASH_RE)
     if pattern.match(text):
         return text
+    elif hash_pattern.match(text):
+        return text
     else:
-        raise InputException("Invalid version number")
+        raise InputException("Invalid version or hash pattern")
```

Comment:
I think these should be separate functions since we don't have any commands which would take either a version number or a commit hash. As far as I can tell the `hotfix` command above should only take commit hashes.